scripts/Audio/player_audio_manager.gd

```diff
@@ -49,6 +49,21 @@ func set_weapon_sfx(weapon_sfx: String) -> void:
 	if weapon_firing_sfx.has_stream_playback():
 		weapon_firing_sfx.get_stream_playback().switch_to_clip_by_name(current_weapon_sfx)
 
+func _on_player_controller_on_moth_jump() -> void:
+	moth_jump_sfx.play()
+
+## When the moth jump is released, switch the active clip to wind down the sound effect. 
+func _on_player_controller_on_moth_jump_ended() -> void:
+	# TODO remove this temporary fix and un-comment the line below
```

Comment:
Can we remove this fix?

Reply:
We're leaving it as is for now. This comment is an artifact from when we were discussing a "better fix"

Reply:
In the player movement controller you can also add a signal to differentiate the dip ending and the moth jump ending easily to get rid of this code. Or you can just remove the signal from the dip also. 

![image](https://github.com/user-attachments/assets/1886a83c-37a9-4ede-9977-acba95d20843)

The two signals being output are shown in the pic I believe